README.md

```diff
@@ -204,11 +203,11 @@ If you happen to have APNS files in `pkcs12` format (.p12 or .pfx extenstion) yo
 
 
 
-## REST API
+## RESTful API
 
 ### Swagger
 
-If for some reason you need `Swagger` spec for this `REST` service, there is swagger endpoint available at `REST` path `/swagger.json`
+If for some reason you need `Swagger` spec for this `RESTful` service, there is a swagger endpoint available at `HTTP` path `/swagger.json`
```

Comment:
If for some reason you need a `Swagger` spec for this `RESTful` service, there is a swagger endpoint available on an `HTTP` path `/swagger.json`
??